pom.xml

```diff
@@ -569,6 +569,22 @@
   <build>
     <defaultGoal>install</defaultGoal>
     <plugins>
+      <plugin>
+        <groupId>org.owasp</groupId>
+        <artifactId>dependency-check-maven</artifactId>
+        <version>3.1.1</version>
```

Comment:
Can you move in the properties the version number, thanks

Reply:
of course, good point